R/facet-.r

```diff
@@ -591,7 +600,7 @@ combine_vars <- function(data, env = emptyenv(), vars = NULL, drop = TRUE) {
     if (drop) {
       new <- unique_combs(new)
     }
-    base <- unique(rbind(base, df.grid(old, new)))
+    base <- unique(vec_rbind(base, df.grid(old, new)))
```

Comment:
Might be worth switching to `vec_unique()` here; I think it's quite a bit faster?

Reply:
For example

``` r
time <- nycflights13::flights[c("year", "month", "day")]

bench::mark(unique(time), vctrs::vec_unique(time))
#> Warning: Some expressions had a GC in every iteration; so filtering is disabled.
#> # A tibble: 2 × 6
#>   expression                   min   median `itr/sec` mem_alloc `gc/sec`
#>   <bch:expr>              <bch:tm> <bch:tm>     <dbl> <bch:byt>    <dbl>
#> 1 unique(time)            401.23ms 428.22ms      2.34    9.22MB    15.2 
#> 2 vctrs::vec_unique(time)   3.64ms   4.73ms    164.      3.31MB     3.99
```

<sup>Created on 2022-06-15 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
nice - is `vec_unique()` one of those vctrs functions where it makes sense to switch 100% to?